src/main/java/uk/co/fivium/els/categories/tumbledryers/model/AirVentedTumbleDryersForm.java

```diff
@@ -0,0 +1,94 @@
+package uk.co.fivium.els.categories.tumbledryers.model;
+
+import javax.validation.constraints.Digits;
+import javax.validation.constraints.NotBlank;
+import uk.co.fivium.els.model.FieldPrompt;
+
+public class AirVentedTumbleDryersForm {
```

Comment:
Air vented and gas-fired tumble dryers look like they have the same fields so they can share the same Form object.
The Condenser dryers also have the same fields with one extra specific field, so the condenser form can `extend` the standard dryer form with the one specific field added.